Scripts/sh/buildAllTools.sh

```
@@ -0,0 +1,11 @@
+#!/usr/bin/env sh
+
+# make sure to start from script dir
+if [ "$(dirname $0)" != "." ]; then
+    cd "$(dirname $0)"
```

Shellcheck

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)